installing_on_microsoft_azure/_topics/installation.md

```diff
@@ -351,7 +351,7 @@ using the Azure Command-Line Interface (Azure CLI).
     minutes. Note: Enter `az storage container list` to get the list of
     storage containers.
 
-        $ az storage blob upload --account-name <storage-account-name> --container-name <container-name> --type page --file <path-to-vhd> --name <image-name>.vhd
+        $ az storage blob upload --account-name <storage-account-name> --container-name <storage-container-name> --type page --file <path-to-vhd> --name <image-name>.vhd
```

Comment:
This change is just for backreference to the change below so it makes more sense.